### DISCUSSION TITLE
issues with "View Once" feature, stickers, and voice notes (2024-09-16)

### DISCUSSION THREAD

**Discussion by ProdByGR** [Schwartzblat/WhatsApp-Web-Plus#24] [Q&A] [answered]
It seems like the "View Once" feature has stopped working again as of 2024-09-16. Additionally, I'm unable to send stickers or voice notes while the extension is enabled. Is anyone else experiencing this issue?

**Comment by Schwartzblat** [comment]
I am working on that. Thanks for the report.

**Comment by Schwartzblat** [comment, marked as answer]
Fixed.
